templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -513,10 +519,9 @@ Parameters:
     Description: Controls whether the load balancer should be visible to the internet (true) or only within the VPC (false).
     Type: String
   KeyPairName:
-    Default: ''
     ConstraintDescription: Must be the name of an existing EC2 Key Pair.
     Description: The EC2 Key Pair to allow SSH access to the instances
-    Type: String
+    Type: AWS::EC2::KeyPair::KeyName
```

Comment:
Previously it was possible to leave this one empty and then the key was sourced from `ATL-DefaultKey` export. If we are removing this option of importing key pair, we should also remove `KeyProvided` Condition https://github.com/atlassian/quickstart-atlassian-jira/blob/39202054ceba1200f59ceecd8e1acaf31b729461/templates/quickstart-jira-dc.template.yaml#L583

I agree it is a better form experience when you have the dropdown for the key pair just we are now making it mandatory.

Reply:
Yep, you're right. This is also in the BB template, I'll remove it from there too.